blog/user-guide.md

```diff
@@ -639,15 +639,15 @@ We value your feedback and are committed to continually improving Magistrala to
 
 IIoT, Open Source, IoT Platform, Industrial Automation, Remote Monitoring, MQTT, CoAP, WebSocket, Device Management, Scalability, Security, Docker, Go, Multi-Tenancy, Data Processing, Real-Time Analytics, Edge Computing, Sensor Networks, Industrial IoT, Connected Devices, IoT Middleware, Protocol Bridging, Cloud Platform, IoT Security, IoT Data Management, Customer Support, Community Engagement, Open Source Community, IIoT Support, Developer Resources
 
-[architecture]: https://docs.magistrala.abstractmachines.fr/architecture/
-[authorization]: https://docs.magistrala.abstractmachines.fr/authorization/#domain-viewer-with-channel-thing
-[storage]: https://docs.magistrala.abstractmachines.fr/storage/#readers
+[architecture]: https://docs.magistrala.absmach.eu/architecture/
```

Comment:
This link does not exist, but this does https://docs.magistrala.absmach.eu/user-guide/architecture.